.changeset/shaggy-pens-rescue.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
This is not what the release is for. Consumers don't care about chromatic stories. 
In general you wouldn't need a release for this. But i'm guessing you are releasing so that the fixes in the code such as dialog and index.css are released. So list those changes instead